Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -795,6 +795,16 @@ public void ReadXmlPrototype(XmlReader readerParent)
                     break;
             }
         }
+
+        if (orderActions.ContainsKey("Uninstall"))
+        {
+            InventoryCommon asInventory = new InventoryCommon();
+            asInventory.type = Type;
+            asInventory.maxStackSize = 1;
+            asInventory.basePrice = 0f;
+            asInventory.category = "crated_furniture";
+            PrototypeManager.Inventory.Add(asInventory);
+        }
```

Comment:
This is what actually creates the Inventory prototype (and sets some baseline info). It could *probably* exist in Uninstall.Initialize, but wasn't put there simply because the Uninstall orderAction didn't exist at the time I made this piece of code.

The one potential sticking point I see is basePrice, I want to have an actual base price set in xml (rather than always being 0), and it feels unintuitive to have it exist in the uninstall tag. I'm not entirely sure how I could get that info to the Uninstall orderAction without having it a property in the orderAction tag, though I will look into seeing what I can do on that front.